### PR TITLE
Throw original exception when command.close() throws Exception too

### PR DESCRIPTION
**Pull request by marcuslinke**

### CODE REVIEW

**Comment by KostyaSha** [comment]
LOL :) just opened IDE to fix it also, please wait for reviews before merge...

File: src/main/java/com/github/dockerjava/api/command/DockerCmd.java [hunk 1]
```diff
@@ -1,7 +1,8 @@
 package com.github.dockerjava.api.command;
 
-import java.io.Closeable;
+public interface DockerCmd<RES_T> extends AutoCloseable {
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/command/DockerCmd.java]: could you describe why Closeable need to be changed to AutoClosable when
`public interface Closeable extends AutoCloseable {` ?

File: src/main/java/com/github/dockerjava/jaxrs/AbstrSyncDockerCmdExec.java [hunk 2]
```diff
@@ -19,25 +19,17 @@ public AbstrSyncDockerCmdExec(WebTarget baseResource) {
...
-            result = execute(command);
-
-        } catch (ProcessingException e) {
-            if (e.getCause() instanceof DockerException) {
-                throw (DockerException) e.getCause();
-            } else {
-                throw e;
-            }
-        } finally {
+        try(CMD_T cmd = command) {
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/jaxrs/AbstrSyncDockerCmdExec.java]: `try (` can i contribute checkstyle plugin with checkstyle rules? :)

File: src/main/java/com/github/dockerjava/api/command/DockerCmd.java (hunk 1 above)
Comment (marcuslinke) [inline src/main/java/com/github/dockerjava/api/command/DockerCmd.java]: I can't ;-) Will fix it. Thanks!

File: src/main/java/com/github/dockerjava/jaxrs/AbstrSyncDockerCmdExec.java (hunk 2 above)
Comment (marcuslinke) [inline src/main/java/com/github/dockerjava/jaxrs/AbstrSyncDockerCmdExec.java]: Maybe I should configure code automatic formatting on save in eclipse...

**Pull request closed without merging**